jdi-light/src/main/java/com/epam/jdi/light/logger/JDILogger.java

```diff
@@ -99,9 +101,9 @@ public void logOff(JAction action) {
         logLevel.set(tempLevel);
         return result;
     }
-    private String name;
-    private Logger logger;
-    private List<Long> multiThread = new ArrayList<>();
+    private final String name;
+    private final Logger logger;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6346938)